configs/kubernetes-rules-configmap.yaml

```diff
@@ -1269,3 +1275,13 @@ data:
           branch: master
         - repository: client-go
           branch: master
+    - destination: component-base
+      library: true
+      branches:
+      - source:
+          branch: master
+          dir: staging/src/k8s.io/component-base
+        name: master
+        dependencies:
+        - repository: apimachinery
```

Comment:
it also depends on client-go and api.

Reply:
Same question as above ^, right now in the PR component-base only depends on apimachinery: https://github.com/kubernetes/kubernetes/blob/66d879c94637489c03d16651ba01a15e60b6116a/staging/src/k8s.io/component-base/Godeps/Godeps.json.

Reply:
@sttts shall we wait until it actually depends on those two as well? or is this just (again) about publishing order?

Reply:
I have added here as well, but let me know if you'd like to wait.

> or is this just (again) about publishing order?

Looking at the code, it looks like the list of dependencies are just used to remove the values for those deps in `Godeps/Godeps.json`, and then regenerate them. If it doesn't find a dep, it'll be a no-op. 

https://github.com/kubernetes/publishing-bot/blob/54cd0ea15d70bd75141fc02405f2d8203fd3fbfa/artifacts/scripts/util.sh#L815-L819

But let's wait for @sttts to confirm this. :)

Reply:
see https://github.com/kubernetes/publishing-bot/pull/142#discussion_r245485117